tests/testthat/test-kwic.R

```diff
@@ -396,3 +396,32 @@ test_that("deprecated keywords argument still works", {
         "fox"
     )
 })
+
+test_that("test docvar is passed through kwic()", {
+    
+    # kwic only copy original docvars
+    testkwic <- kwic(data_corpus_inaugural, 'american')
+    expect_equal(attr(testkwic, 'docvars'),
```

Comment:
I suggest you write an internal `docvars.kwic()` and use that to do the multiplication of rows of docvars in `as.tokens()`. 

We could use this for the `corpus.kwic()` as well but we would need to add options to make it possible to do it as one keyword per document with the pre/keyword/post as a document, versus the current approach of having 2*(number of keywords) as corpus documents, with a variable for pre v. post.